src/C++/FieldConvertors.h

```diff
@@ -216,10 +216,7 @@ struct CheckSumConvertor
   {
     if ( value > 255 || value < 0 ) throw FieldConvertError();
     char result[3];
-    if( integer_to_string_padded(result, sizeof(result), value) != result )
```

Comment:
This does not need to be checked as the function integer_to_string_padded is now being tested by its own unit test in FieldConvertorsTestCase.cpp.  This new unit test ensures the function does not produce the behavior being checked by this condition.